embark/uploader/views.py

```diff
@@ -318,6 +318,7 @@ def html_report(request, analyze_id, hmtl_file):
     report_path = Path(f'/app/emba{request.path}')
 
     html_body = get_template(report_path)
+    logger.info("html_report - analyze_id: %s hmtl_file: %s", analyze_id, hmtl_file)
```

Comment:
typo

Reply:
hmtl_file? It is also in the function def ... so, I have not changed it for now.

Reply:
I think it is a typo and both should be renamed to `html_file`.